src/modules/core/components/CalloutCard/CalloutCard.md

```diff
@@ -0,0 +1,61 @@
+**Description**
```

Comment:
Very cool of you to add a styleguide entry for this, but I think we're going to discontinue this. _(unless you guys have other opinions on this)_

Leave it in for now